pkg/v1/providers/infrastructure-aws/v1.2.0/cconly/base.yaml

```diff
@@ -987,16 +1059,16 @@ spec:
               cloud-provider: aws
               tls-cipher-suites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
           dns:
-            imageRepository: projects-stg.registry.vmware.com/tkg #! TODO '${_TKG_COREDNS_IMAGE_REPOSITORY}' May be a variable for package level
-            imageTag: v1.8.4_vmware.5 #! TODO '${_TKG_COREDNS_IMAGE_TAG}' May be a variable for package level
+            imageRepository: projects-stg.registry.vmware.com/tkg
```

Comment:
nit: populating with real looking but likely incorrect values only to be patched is misleading. Can we not set it at all? If we want to suggest example values, perhaps do something like:
```
imageTag: #! e.g v3.5.0_vmware.5
```